examples/repl/cmdline.rs

```diff
@@ -141,35 +141,23 @@ unsafe fn poke_spec(context: &Context, spec: *const libc::c_char) -> libc::c_int
         ok_to_continue = true;
     } else {
         let rs = context.sql.get_config(context, "import_spec");
-        if rs.is_none() {
-            error!(context, 0, "Import: No file or folder given.");
```

Comment:
This error is helpful in my opinion, we shouldn't drop it.

Reply:
removed the renaming commit that contained it